CONTRIBUTING.md

```diff
@@ -16,6 +16,14 @@ We use [`rake`](https://github.com/ruby/rake) for task automation.
 - iOS 13 or higher
 - Xcode 14 or higher
 
+Then get certs and profiles:
+```shell
+bundle exec fastlane run get_certificates
+bundle exec fastlane match development --readonly
+bundle exec fastlane match adhoc --readonly
```

Comment:
I think this is redundant since we only have provisioning profiles for two environments: development and distribution.